### PR TITLE
https://issues.redhat.com/browse/ACM-14521

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira for including sample: https://issues.redhat.com/browse/ACM-14521 

See Jira for what's new entry: https://issues.redhat.com/browse/ACM-14306

### CODE REVIEW

File: business_continuity/backup_restore/backup_restore.adoc [hunk 1]
```diff
@@ -276,11 +276,37 @@ spec:
...
-* The `restore.cluster.open-cluster-management.io` resource is still active if the phase is `Enabled` or `Running`. The resource execution is completed if the phase changes to `Finished`. After the restore operation is completed, you can run another restore operation on the same hub cluster. 
-* Only one `restore.cluster.open-cluster-management.io` can run at one time. 
+**  The `restore.cluster.open-cluster-management.io` resource is still active if the phase is `Enabled` or `Running`. The resource execution is completed if the phase changes to `Finished`. After the restore operation is completed, you can run another restore operation on the same hub cluster. 
+** Only one `restore.cluster.open-cluster-management.io` can run at one time. 
+
+
+* Use the advanced options with velero restore to filer out resources you want to restore. Use the following sample with velero restore spec options to restore resources only from the `vb-managed-cis-2` managed cluster namespace and to exclude global `MultiCluster` resources: 
+
++
+[source,yaml]
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_restore.adoc:290]: The sample should be ( please make sure the indentation is correct .. ) 


apiVersion: cluster.open-cluster-management.io/v1beta1
kind: Restore
metadata:
  name: restore-filter-sample
  namespace: open-cluster-management-backup
spec:
  cleanupBeforeRestore: None 
  veleroCredentialsBackupName: latest 
  veleroResourcesBackupName: latest  
  veleroManagedClustersBackupName: latest 
  excludedResources:
    - ManagedCluster
  orLabelSelectors:
    - matchExpressions:
        - values:
            - vb-managed-cls-2
          key: name
          operator: In

File: business_continuity/backup_restore/backup_restore.adoc [hunk 2]
```diff
@@ -276,11 +276,37 @@ spec:
...
+spec:
+  includedResources
+  includedNamespaces
+  excludedResources
+  excludedNamespaces
+  LabelSelector
+  OrLabelSelector 
+----
++
+
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_restore.adoc:326]: Add this section : 

You can set the following velero restore spec options when creating the ACM restore resource:
  includedResources
  includedNamespaces
  excludedResources
  excludedNamespaces
  LabelSelector
  OrLabelSelector

**Review by birsanv — CHANGES_REQUESTED** [review]
please update sample and add the list of filters

**Review by birsanv — APPROVED** [review]
lgtm

**Review by birsanv — CHANGES_REQUESTED** [review]
sorry, need to make requested updates

**Review by birsanv — APPROVED** [review]
lgtm

File: business_continuity/backup_restore/backup_restore.adoc [hunk 3]
```diff
@@ -276,11 +276,56 @@ spec:
   veleroCredentialsBackupName: skip
   veleroResourcesBackupName: skip
 ----
++
 
 *Notes:*
 
-* The `restore.cluster.open-cluster-management.io` resource is still active if the phase is `Enabled` or `Running`. The resource execution is completed if the phase changes to `Finished`. After the restore operation is completed, you can run another restore operation on the same hub cluster. 
-* Only one `restore.cluster.open-cluster-management.io` can run at one time. 
+**  The `restore.cluster.open-cluster-management.io` resource is still active if the phase is `Enabled` or `Running`. The resource execution is completed if the phase changes to `Finished`. After the restore operation is completed, you can run another restore operation on the same hub cluster.
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_restore.adoc]: Can we use a different word than "execution"?

File: business_continuity/backup_restore/backup_restore.adoc [hunk 4]
```diff
@@ -276,11 +276,56 @@ spec:
...
 
 *Notes:*
 
-* The `restore.cluster.open-cluster-management.io` resource is still active if the phase is `Enabled` or `Running`. The resource execution is completed if the phase changes to `Finished`. After the restore operation is completed, you can run another restore operation on the same hub cluster. 
-* Only one `restore.cluster.open-cluster-management.io` can run at one time. 
+**  The `restore.cluster.open-cluster-management.io` resource is still active if the phase is `Enabled` or `Running`. The resource execution is completed if the phase changes to `Finished`. After the restore operation is completed, you can run another restore operation on the same hub cluster. 
+** Only one `restore.cluster.open-cluster-management.io` can run at one time. 
+
+
+* Use the advanced options with velero restore to filer out resources you want to restore. Use the following sample with velero restore spec options to restore resources only from the `vb-managed-cis-2` managed cluster namespace and to exclude global `MultiCluster` resources:
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_restore.adoc]: Is velero supposed to be capital ? Or are you referencing a resource? spec should also be monospaced or you should write out specification

File: business_continuity/backup_restore/backup_restore.adoc [hunk 5]
```diff
@@ -276,11 +276,56 @@ spec:
...
+  - ManagedCluster
+orLabelSelectors:
+  - matchExpressions:
+  - values:
+  - vb-managed-cls-2
+key: name
+operator: In
+----
+
+* Set the following velero restore spec options when you create the {acm-short} restore resource:
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_restore.adoc]: Same thing here, is velero suppose to be capital?

File: business_continuity/backup_restore/backup_restore.adoc [hunk 6]
```diff
@@ -276,11 +276,56 @@ spec:
...
++
+[source,yaml]
+----
+includedResources
+includedNamespaces
+excludedResources
+excludedNamespaces
+LabelSelector
+OrLabelSelector
+----
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_restore.adoc:324]: Should `spec` be added to this YAML snippet?

File: business_continuity/backup_restore/backup_restore.adoc [hunk 7]
```diff
@@ -276,11 +276,56 @@ spec:
...
++
+
+*Note:*
+
+** When you set any of these velero restore filters, set the `cleanupBeforeRestore` to `None` to avoid cleaning up hub resources that are part of the restored backup but are not being restored because of the applied filter.
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_restore.adoc]: Should velero be capitalized. Reminder to write hub cluster

File: release_notes/acm_whats_new.adoc [hunk 8]
```diff
@@ -108,6 +108,8 @@ See link:../governance/grc_intro.adoc#governance[Governance] to learn more about
 
 * You can now keep the primary hub cluster active during a restore operation. See link:../business_continuity/backup_restore/backup_active_restore.adoc#keep-hub-active-restore[Keeping the primary hub cluster active during a restore process].
 
+* With {acm-short} restore resources, you can set more velero restore spec options. See link:../business_continuity/backup_restore/backup_restore.adoc#more-restore-samples[Using other restore samples].
```
Comment (dockerymick) [inline release_notes/acm_whats_new.adoc]: Since you are mentioning resources, should restore be capitalized?

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: business_continuity/backup_restore/backup_restore.adoc (hunk 4 above)
Comment (birsanv) [reply business_continuity/backup_restore/backup_restore.adoc]: we can say velero.io.restore

File: business_continuity/backup_restore/backup_restore.adoc (hunk 5 above)
Comment (birsanv) [reply business_continuity/backup_restore/backup_restore.adoc]: we can say velero.io.restore

File: business_continuity/backup_restore/backup_restore.adoc (hunk 7 above)
Comment (birsanv) [reply business_continuity/backup_restore/backup_restore.adoc]: yes, Velero is the product name

File: business_continuity/backup_restore/backup_restore.adoc [hunk 9]
```diff
@@ -280,11 +280,11 @@ spec:
...
+**  The `restore.cluster.open-cluster-management.io` resource is still active if the phase is `Enabled` or `Running`. The resource run is completed if the phase changes to `Finished`. After the restore operation is completed, you can run another restore operation on the same hub cluster. 
 ** Only one `restore.cluster.open-cluster-management.io` can run at one time. 
 
 
-* Use the advanced options with velero restore to filer out resources you want to restore. Use the following sample with velero restore spec options to restore resources only from the `vb-managed-cis-2` managed cluster namespace and to exclude global `MultiCluster` resources: 
+* Use the advanced options with `velero.io.restore` to filer out resources you want to restore. Use the following sample with `velero.io.restore` spec options to restore resources only from the `vb-managed-cis-2` managed cluster namespace and to exclude global `MultiCluster` resources:
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_restore.adoc]:
```suggestion
* Use the advanced options with `velero.io.restore` to filer out resources you want to restore. Use the following sample with `velero.io.restore` `spec` options to restore resources only from the `vb-managed-cis-2` managed cluster namespace and to exclude global `MultiCluster` resources: 
```

File: business_continuity/backup_restore/backup_restore.adoc [hunk 10]
```diff
@@ -309,23 +309,24 @@ key: name
 operator: In
 ----
 
-* Set the following velero restore spec options when you create the {acm-short} restore resource: 
+* Set the following `velero.io.restore` spec options when you create the {acm-short} restore resource:
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_restore.adoc]:
```suggestion
* Set the following `velero.io.restore` `spec` options when you create the {acm-short} restore resource: 
```

File: release_notes/acm_whats_new.adoc [hunk 11]
```diff
@@ -108,7 +108,7 @@ See link:../governance/grc_intro.adoc#governance[Governance] to learn more about
 
 * You can now keep the primary hub cluster active during a restore operation. See link:../business_continuity/backup_restore/backup_active_restore.adoc#keep-hub-active-restore[Keeping the primary hub cluster active during a restore process].
 
-* With {acm-short} restore resources, you can set more velero restore spec options. See link:../business_continuity/backup_restore/backup_restore.adoc#more-restore-samples[Using other restore samples].
+* With {acm-short} restore resources, you can set more `velero.io.restore` spec options. See link:../business_continuity/backup_restore/backup_restore.adoc#more-restore-samples[Using other restore samples].
```
Comment (dockerymick) [inline release_notes/acm_whats_new.adoc]:
```suggestion
* With {acm-short} restore resources, you can set more `velero.io.restore` `spec` options. See link:../business_continuity/backup_restore/backup_restore.adoc#more-restore-samples[Using other restore samples].
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: business_continuity/backup_restore/backup_restore.adoc [hunk 12]
```diff
@@ -276,11 +276,57 @@ spec:
...
 
 *Notes:*
 
-* The `restore.cluster.open-cluster-management.io` resource is still active if the phase is `Enabled` or `Running`. The resource execution is completed if the phase changes to `Finished`. After the restore operation is completed, you can run another restore operation on the same hub cluster. 
-* Only one `restore.cluster.open-cluster-management.io` can run at one time. 
+**  The `restore.cluster.open-cluster-management.io` resource is still active if the phase is `Enabled` or `Running`. The resource run is completed if the phase changes to `Finished`. After the restore operation is completed, you can run another restore operation on the same hub cluster. 
+** Only one `restore.cluster.open-cluster-management.io` can run at one time. 
+
+
+* Use the advanced options with `velero.io.restore` to filer out resources you want to restore. Use the following sample with `velero.io.restore` `spec` options to restore resources only from the `vb-managed-cis-2` managed cluster namespace and to exclude global `MultiCluster` resources:
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_restore.adoc]: this is a typo: it should be vb-managed-cls-2, not vb-managed-cis-2

**Review by birsanv — APPROVED** [review]
lgtm

**Review by dockerymick — APPROVED** [review]
Thanks, this looks good to me

**Pull request merged**